.github/workflows/build_pr_documentation.yml

```diff
@@ -28,63 +27,80 @@ jobs:
           repository: 'huggingface/optimum'
           path: optimum
 
-      - uses: actions/setup-node@v2
+      - uses: actions/checkout@v2
         with:
-          node-version: '16'
+          repository: 'huggingface/optimum-habana'
+          path: optimum-habana
 
-      - name: Set env
+      - name: Set environment variables
         run: |
-          echo "WRITE=$(echo 'ghp_'$(wget -qO- lysand.re/doc-build-dev)'bm')" >> $GITHUB_ENV
+          echo "write_token=$(echo 'ghp_'$(wget -qO- lysand.re/doc-build-dev)'bm')" >> $GITHUB_ENV
 
       - name: Setup environment
         run: |
           pip uninstall -y doc-builder
           cd doc-builder
           git pull origin main
-          pip install -e .
+          pip install .
+          pip install black
           cd ..
 
+      - name: Create build directory
+        run: |
+          mkdir build_dir
+          mkdir build_dir/optimum
+          cd build_dir/optimum
+          wget https://raw.githubusercontent.com/huggingface/doc-build/main/optimum/_versions.yml
+
+      - name: Make Habana documentation
+        run: |
+          cd optimum-habana
+          git fetch origin
```

Comment:
These lines should be removed when https://github.com/huggingface/optimum-habana/pull/68 is merged

Reply:
This PR has been merged!